src/Console/Command/ReleaseCommand.php

```diff
@@ -0,0 +1,289 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DevKit\Console\Command;
+
+use Github\Exception\ExceptionInterface;
+use Packagist\Api\Result\Package;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Jordi Sala <jordism91@gmail.com>
+ */
+final class ReleaseCommand extends AbstractCommand
+{
+    private static $labels = array(
+        'patch' => 'blue',
+        'bug' => 'red',
+        'docs' => 'yellow',
+        'minor' => 'green',
+        'pedantic' => 'cyan',
+    );
+
+    private static $stabilities = array(
+        'patch' => 'blue',
+        'minor' => 'green',
+        'stable' => 'yellow',
```

Comment:
I would like a word for PRs that does not require a release. We could use pedantic, but then docs PR should get this pedantic status.

Summary:

Given a currentRelease 3.3.0 and a Pull request with the following status:

'patch' => will result in 3.3.1
'minor' => will result in 3.4.0
'????' => will result in 3.3.0 (no release needed)
'major' => will result in 4.0.0 (not the scope of this PR)

What word should I use? @greg0ire @OskarStark